Godot_test/ClientScripts/EquipmentItem.cs

```diff
@@ -0,0 +1,28 @@
+using Godot;
+using Godot.Collections;
+using Roguelike.Tiles.Looting;
+using System;
+
+public partial class EquipmentItem : Node
+{
+	public Roguelike.Tiles.Looting.Equipment equipment;
+	private string pathToEquipment = "res://Sprites/Loot/Equipment/";
+	private Dictionary<Weapon.WeaponKind, string> weaponKindPaths = new Dictionary<Weapon.WeaponKind, string> { { Weapon.WeaponKind.Sword, "Swords/rusty_sword.png" }, { Weapon.WeaponKind.Bashing, "Bashing/branch.png" }, { Weapon.WeaponKind.Axe, "Axes/axe.png" }, { Weapon.WeaponKind.Dagger, "Daggers/dagger.png" } };
```

Comment:
make a new line after each element so that horiz. scrollbar is not needed to review code